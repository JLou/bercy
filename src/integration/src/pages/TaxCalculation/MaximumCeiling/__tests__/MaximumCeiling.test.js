import React from 'react';
import { render } from '@testing-library/react';
import { MaximumCeiling } from '../MaximumCeiling';

describe('MaximumCeiling', () => {
  it('Renders MaximumCeiling component without crashing', () => {
    const { asFragment } = render(<MaximumCeiling />);
    expect(asFragment()).toMatchSnapshot();
  });
});
