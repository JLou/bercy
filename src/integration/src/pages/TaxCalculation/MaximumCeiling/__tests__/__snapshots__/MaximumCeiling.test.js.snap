// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MaximumCeiling Renders MaximumCeiling component without crashing 1`] = `
<DocumentFragment>
  <article
    class="af-panel"
  >
    <header
      class="af-panel__header"
    >
      <span
        class="af-panel__title"
      >
        Taux d'imposition par année
      </span>
      <div
        class="af-form__select-container"
      >
        <select
          class="af-form__input-select"
          name="name"
        >
          <option
            value="choice1"
          >
            2019
          </option>
          <option
            value="choice2"
          >
            2020
          </option>
        </select>
        <span
          class="glyphicon glyphicon-menu-down"
        />
      </div>
    </header>
    <section
      class="af-panel__content"
    >
      <table
        class="af-table"
      >
        <thead
          class="af-table-thead"
        >
          <tr
            class="af-table__tr"
          >
            <th
              class="af-table__th"
            >
              <span
                class="af-table-th-content"
              >
                Tranches
              </span>
            </th>
            <th
              class="af-table__th"
            >
              <span
                class="af-table-th-content"
              >
                Taux d'imposition
              </span>
            </th>
          </tr>
        </thead>
        <tbody
          class="af-table__body"
        >
          <tr
            class="af-table__tr"
          >
            <td
              class="af-table__cell"
            >
              <span
                class="af-table-body-content"
              >
                jusqu'à 9 964€
              </span>
            </td>
            <td
              class="af-table__cell"
            >
              <b>
                0%
              </b>
            </td>
          </tr>
          <tr
            class="af-table__tr"
          >
            <td
              class="af-table__cell"
            >
              <span
                class="af-table-body-content"
              >
                de 9964 à 27519
              </span>
            </td>
            <td
              class="af-table__cell"
            >
              <b>
                14%
              </b>
            </td>
          </tr>
        </tbody>
      </table>
    </section>
  </article>
</DocumentFragment>
`;
